index.html

```diff
@@ -54,8 +54,97 @@ <h2>LET'S CONNECT</h2>
     </section>
   </header>  
    
-</body>
-</html>
-
+  <section class="work">
+    <article class="card-container">
+      <div class="gb-img">
+        <img src="img/Snapshoot Portfolio.png">
+      </div>
+      <div class="contents">
+        <div class="heading">
+          <h3>Tonic</h3>
+        </div>
+        <div class="skills">
+          <ul>
+            <li>
+              <span class="canopy">Canopy</span>
+            </li>
+            <li>
+              <div class="div"></div>
+            </li>
+            <span>Full Stack Dev</span>
+            <li>
+              <div class="div"></div>
+            </li>
+            <span>2015</span>
+          </ul>
+        </div>
+        <p class="heading2">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <div class="tech">
+          <ul>
+            <li><span>HTML</span> </li>
+            <li><span>CSS</span> </li>
+            <li><span>Javascript</span> </li>
+          </ul>
+        </div>
+          <button type="submit" class="btn-a ">See Projects</button>
+      </div>
+    </article>
+  </section>
 
+  <!-- about section -->
+  <section class="about-container">
+    <article class="about">
+        <h1 class="about-h1-b">About Myself</h1>
+        
+        <h3 class="about-p-b">
+          I'm a Software developer! Can help you build a product, feature or Website Look through Some of my Work and
+          experience! if you like What you see and have a project you need coded, don't hesitate to contact me.
+        </h3>
+        
+        <h3 class="connect-b">LET'S CONNECT</h3>
+        
```

Comment:
- [ ] Lastly, I noticed that after implementing the changes requested, some parts of your design have changed and don't look quite similar to the Figma design :+1: 
- From the images below, you can see that in your design, at the end of the works section, and on top of the About section, there is a square rectangular shape that is currently disrupting the design and not making it look like the Figma design. So kindly remove this from your design so your design looks exactly like the Figma design :wink: 

| Your Design | The Figma Design |
|--------|--------|
| ![image](https://github.com/Isaac-ej/Portfolio_Website/assets/102699958/bd5b60aa-d00e-4974-95b6-5fb94086ee46) | ![image](https://github.com/Isaac-ej/Portfolio_Website/assets/102699958/fffac147-7136-4edc-ab3d-f5a296368187) |